src/check-update.ts

```diff
@@ -4,12 +4,14 @@ import {
   DENO_SLACK_SDK,
 } from "./libraries.ts";
 
+import { getJSON } from "./utilities.ts";
```

Comment:
Our first shared utility class! 🥲 I wasn't sure where to put him, so happy to move him elsewhere if folks have opinions on this.